crd/nodenetworkconfig/api/v1alpha/groupversion_info.go

```diff
@@ -1,19 +1,3 @@
-/*
```

Comment:
I am by no means a legal expert but do we want to keep the licenses of generated files as the license that they were generated under?

Reply:
These files are generated by kubebuilder as apache2 by default, but that's customizable and we should have overriden it when we scaffolded the CRDs. Since currently that's the default and these are the only files not covered by the root MIT license, and that seemed accidental. Removing the headers covers them under MIT with everything else.